File: exams/u2/padilla/Padilla 61-70/Padilla 61-70/adrizzx/Q61-70/StudentManagement/src/main/java/ec/edu/espe/studentmanagement/controller/MongoDBConnection.java

package ec.edu.espe.studentmanagement.controller;

/**
 *
 * @author Adrian Padilla
 */
import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoDatabase;

public class MongoDBConnection {
    private static final String CONNECTION_STRING = "mongodb+srv://mapadilla12:82410Loveadrian.@adrianpadilla.b8xh14p.mongodb.net/";
    private static MongoClient mongoClient;
    private static MongoDatabase database;

    public static void connect() {
        mongoClient = MongoClients.create(CONNECTION_STRING);
        database = mongoClient.getDatabase("students_db"); // Cambia "students_db" por el nombre de tu base de datos
    }

    public static MongoDatabase getDatabase() {
        return database;
    }
}

File: ws/sigcha/u3/ws35ExamInOurLanguage/ActivityPairs/controller/mongodb_connection.py
from pymongo import MongoClient

class MongoDBConnection:
    CONNECTION_STRING = "mongodb+srv://kjsigcha:kjsigcha@cluster0.j5elpou.mongodb.net/"
    mongo_client = None
    database = None

    @staticmethod
    def connect():
        MongoDBConnection.mongo_client = MongoClient(MongoDBConnection.CONNECTION_STRING)
        MongoDBConnection.database = MongoDBConnection.mongo_client["buildings_db"]

    @staticmethod
    def get_database():
        return MongoDBConnection.database